gojmx/types.go

```diff
@@ -57,6 +62,38 @@ func (j *AttributeResponse) GetValue() interface{} {
 	}
 }
 
+// GetValueAsFloat casts the value from AttributeResponse to float based on type.
+func (j *AttributeResponse) GetValueAsFloat() (float64, error) {
+	switch (*j).ResponseType {
+	case ResponseTypeBool:
+		if j.BoolValue {
+			return 1, nil
+		}
+		return 0, nil
+	case ResponseTypeString:
+		parsedValue, err := strconv.ParseFloat(fmt.Sprintf("%v", j.StringValue), 64)
+		if err != nil {
+			return 0, err
+		}
+		if isNaNOrInf(parsedValue) {
```

Comment:
i think Nan values/ inf etc are handled by the SDK. Library can return them

Reply:
Ok, i'll remove this.

Reply:
Fixed in https://github.com/newrelic/nrjmx/pull/130/commits/eea0462dc1a5b686750c4c07de429a62606a5996